### ISSUE TITLE
Nested objects that aren't arrays do not insert correctly

### ISSUE THREAD

**Issue by adam-charlton** [mrjono1/joi-to-typescript#2]
When generating types from the following schemas the nested object is not correctly referenced.

```
import Joi from 'joi';

export const BarSchema = Joi.object({
  id: Joi.number().required().description('Id').example(1)
}).label('Bar');

export const FooSchema = Joi.object({
  id: Joi.number().required().description('Id').example(1),
  bar: BarSchema.required().description('Bar')
}).label('Foo');
```

Results in the below, with object being used instead of Bar.

```
/**
 * This file was automatically generated by joi-to-typescript
 * Do not modify this file manually
 */

import { object } from '.';

/**
 * Bar
 */
export interface Bar {
  /**
   * id
   * Id
   */
  id: number;
}

/**
 * Foo
 */
export interface Foo {
  /**
   * bar
   * Bar
   */
  bar: object;
  /**
   * id
   * Id
   */
  id: number;
}

```

**Comment by mrjono1** [comment]
https://github.com/mrjono1/joi-to-typescript/pull/3
Thanks for the issue, I have greatly improved the tests as a result of this find

**Comment by mrjono1** [comment]
Fix has been released on version v1.3.1
